clerk/organizations.go

```diff
@@ -59,6 +59,24 @@ func (s *OrganizationsService) Update(organizationID string, params UpdateOrgani
 	return &organization, nil
 }
 
+type UpdateMetadataParams struct {
+	PublicMetadata  json.RawMessage `json:"public_metadata"`
+	PrivateMetadata json.RawMessage `json:"private_metadata"`
+	OrganizationID  string          `json:"-"`
+}
+
+func (s *OrganizationsService) UpdateMetadata(params UpdateMetadataParams) (*Organization, error) {
```

Comment:
:wrench: Let's have `organizationID` as a separate parameter. This way it conforms to the convention of similar update methods. 

```go
type UpdateMetadataParams struct {
  PublicMetadata json.RawMessage `json:"public_metadata,omitempty"`
  PrivateMetadata json.RawMessage `json:"private_metadata,omitempty"`
}

func (s *OrganizationsService) UpdateMetadata(organizationID string, params UpdateMetadataParams) (*Organization, error) {
```

Reply:
@AlexNti I just noticed, `UpdateMetadataParams` should be renamed to `UpdateOrganizationMetadataParams` since it's in the `clerk` package.

The final API should be
```go
clerk.NewClient().Organizations().UpdateMetadata("org_123", &clerk.UpdateOrganizationMetadataParams{}) 
```